tests/testthat/_snaps/isomap.md

```diff
@@ -27,8 +27,8 @@
     Condition
       Error in `step_isomap()`:
       Caused by error in `prep()`:
-      ! `step_isomap` failed with error:
-      Error : TridiagEigen: eigen decomposition failed
+      x Failed with error:
```

Comment:
```suggestion
```

Reply:
I don't know how easy this is to fix, as it is a sloppy re-throw. I'll add an issue